README.rst

```diff
@@ -609,6 +609,39 @@ Cache can be disabled via ``--using-cache`` option or config file:
         ->setUsingCache(false)
     ;
 
+Cache file can be specified via ``--cache-file`` option or config file:
+
+.. code-block:: php
+
+    <?php
+
+    return Symfony\CS\Config\Config::create()
+        ->setCacheFile(__DIR__.'/.php_cs.cache')
+    ;
+
+Speed up builds on Travis
```

Comment:
Travis is an optional service just like StyleCI. Many people may choose to use other ci providers to run their tests.